service/controller/namespacelabeler/resource/namespacelabel/create.go

```diff
@@ -0,0 +1,40 @@
+package namespacelabel
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/giantswarm/rbac-operator/pkg/label"
+	"github.com/giantswarm/rbac-operator/service/controller/namespacelabeler/key"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	namespace, err := key.ToNamespace(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	if needsUpdate(namespace) {
+		namespace.ObjectMeta.Labels[label.Cluster] = ""
+		namespace.ObjectMeta.Labels[label.Organization] = ""
```

Comment:
Why not setting values from legacy labels?

Reply:
done